### DISCUSSION TITLE
Version 0.7.1

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#629] [Announcements]
Good day! Machinaris v0.7.1 is now available. 

![drives_grid](https://user-images.githubusercontent.com/12936/161400687-12803a56-01c2-4296-b69c-d895efa04378.png)

Changes include:
 - Drive monitoring using Smartctl (WebUI status currently, alerting to come soon)
 - Dutch translations (nl_NL) provided by @bdeprez. Thanks!
 - Updates to various fork blockchains and tools including BTCGreen, Flax, HDDCoin, Madmax, MMX and Shibgreen
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.3.1) - v1.3.1 patch release
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.3.2) - v1.3.2 patch release
  
Notes:
 - Drive monitoring not supported on [Windows](https://github.com/guydavis/machinaris/wiki/Windows) hosts unfortunately due to Docker device pass-thru.
 - Migration to new [Flax](https://github.com/guydavis/machinaris/wiki/Flax) database format requires manual step: `flax db upgrade`

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.7.1'>Version 0.7.1</a>.</em>

**Comment by k2skaterii** [comment]
Hey Guy! 

I'll be tweaking my docker-compose.yaml's to enable drive monitoring.  Excited to see that!  Thank you!

Before I work on the drive monitoring, question about your other note:

> Migration to new Flax database format requires manual step:  `flax db upgrade`

After upgrading to 0.7.1 the first thing I tried was `docker exec machinaris flax db upgrade` - which resulted in `OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "flax": executable file not found in $PATH: unknown`

I tried executing `flax db upgrade` from within the container itself - similar results - `bash: flax: command not found`.

Did I miss something here?

**Reply by k2skaterii** [reply]
Cancel that... I sometimes forget that machinaris supports more than just "Chia".  I read your note and thought it was an upgrade to the "machinaris DB"... Didn't cross my mind that is was an upgrade to the DB used by one of the non-Chia blockchains! 

Feel like a dummy for posting a question.  
Fell like less of a dummy because I figured it out before you (or anyone else) had a chance to answer!
